src/redactable.rs

```diff
@@ -0,0 +1,19 @@
+/// Types that are sensitive data or PII (Personally Identifiable Information) and can be redact-formatted.
+///
+/// The type must have an implementation of [`Display`](std::fmt::Display) so that it can also be formatted in plain text without redaction.
+///
+/// This trait can be derived using the [`Redactable`](derive.Redactable.html) macro.
+pub trait Redactable: std::fmt::Display {
```

Comment:
What is the point of making this a trait? Do we want people to impl it themselves or take it as an argument? Should it be automatically derived from the debug impl when deriving Redact?

Reply:
I think it should be up to them - I aimed to make it as flexible as possible. Basically, "go nuts"

> Should it be automatically derived from the debug impl when deriving Redact?

Nope, since it uses `Display` under the hood, and that might not be implemented